### ISSUE TITLE
Weird box like characters

### ISSUE THREAD

**Issue by oguh43** [arnog/mathlive#2351]
## Description

 I defined a custom keycap like so: `{label: "σ", latex: "\\sqrt{\\frac{1}{#?-1} \\sum_{i=1}^{#?} (#?_i - \\overline{#?})}", class: "tex"}`. Clicking it, produces this weird thing: 
![image](https://github.com/arnog/mathlive/assets/13382592/f77891b7-34a3-4fcc-9d3a-071e97a58c22)
Appending it to DOM via function below produces correct output
```js
mfe.addEventListener("change", ()=>{
        console.log(mfe.value, convertLatexToMarkup(mfe.value))
        history.innerHTML = `<p>${convertLatexToMarkup(mfe.value)}</p>` + history.innerHTML
        mfe.value = ""

    })
```
![image](https://github.com/arnog/mathlive/assets/13382592/d7cf4f92-d4ee-4628-99d0-3855e64e3cdf)
However, as soon as the blank spaces are filled in with values, it once again produces wrong output:
![image](https://github.com/arnog/mathlive/assets/13382592/9b5df902-0b40-44af-a9af-1e7d5d55e5f5)

Also the sqrt looks weird, the top line is offset even as a keycap:
![image](https://github.com/arnog/mathlive/assets/13382592/8dc6e03d-38b6-4186-9798-812fd1b204f1)

Could this be because MathLive fails to load all fonts? 
```
GET file:///E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts/KaTeX_Main-Regular.woff2 net::ERR_FILE_NOT_FOUND
E:\Egithub\IVS_projekt_2\node_modules\mathlive\dist\mathlive.min.js:2813 MathLive 0.98.6: The math fonts could not be loaded from "file:///E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts" {cause: DOMException: A network error occurred.}
Qr @ E:\Egithub\IVS_projekt_2\node_modules\mathlive\dist\mathlive.min.js:2813
await in Qr (async)
connectedCallback @ E:\Egithub\IVS_projekt_2\node_modules\mathlive\dist\mathlive.min.js:3065
(anonymous) @ E:\Egithub\IVS_projekt_2\src\beta_preload.js:27
/E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts/KaTeX_Math-Italic.woff2:1 
        
        
GET file:///E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts/KaTeX_Math-Italic.woff2 net::ERR_FILE_NOT_FOUND
/E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts/KaTeX_Math-BoldItalic.woff2:1 
        
        
 GET file:///E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts/KaTeX_Math-BoldItalic.woff2 net::ERR_FILE_NOT_FOUND
/E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts/KaTeX_SansSerif-Regular.woff2:1 
        
        
 GET file:///E:/Egithub/IVS_projekt_2/node_modules/mathlive/dist/fonts/KaTeX_SansSerif-Regular.woff2 net::ERR_FILE_NOT_FOUND
```
Even though they are present in the specified directories...
### Actual Behavior

> Weird, non-showable characters producing boxes

### Expected Behavior

> Beautifull representation of standard deviation

### Environment

**MathLive version** 
0.98.6
**Operating System**
Windows 11

**Browser** 
Electron - chromium based

**Comment by oguh43** [comment]
The broken characters that are generated are:
Two times
![image](https://github.com/arnog/mathlive/assets/13382592/3f64a59e-3ec0-4398-a3b1-7449f9c5f69d)
One time
![image](https://github.com/arnog/mathlive/assets/13382592/f7733997-a590-4ab3-90ce-616d97535d36)

Charecters in question: `` and ``

**Comment by arnog** [comment]
yes, this is because the required fonts cannot be loaded. Since you are using Electron, you may need to do some specific setups in your environment to allow access to the font files.

**Comment by arnog** [comment]
ntp

**Comment by oguh43** [comment]
Only fonts installed via npm are these: 
![image](https://github.com/arnog/mathlive/assets/13382592/ee07e8f3-d89a-4c63-a8f5-67027e6c0a57)

**Comment by oguh43** [comment]
Npm refused to install all fonts due to a broken external package
